bin/migrate-oats-data/applications/app_prep.py

```diff
@@ -0,0 +1,316 @@
+from common import (
+    ALRChangeCode,
+    AlcsNfuTypeCode,
+    AlcsNfuSubTypeCode,
+    OatsCapabilitySourceCode,
+    OatsAgriCapabilityCodes,
+    AlcsAgCap,
+    AlcsAgCapSource,
+    log_end,
+    log_start,
+)
+from db import inject_conn_pool
+from constants import BATCH_UPLOAD_SIZE
+from psycopg2.extras import execute_batch, RealDictCursor
+import traceback
+from enum import Enum
+
+
+etl_name = "alcs_app_prep"
+
+
+class OatsToAlcsNfuTypes(Enum):
+    AGR = AlcsNfuTypeCode.Agricultural_Farm
+    CIV = AlcsNfuTypeCode.Civic_Institutional
+    COM = AlcsNfuTypeCode.Commercial_Retail
+    IND = AlcsNfuTypeCode.Industrial
+    OTH = AlcsNfuTypeCode.Other
+    REC = AlcsNfuTypeCode.Recreational
+    RES = AlcsNfuTypeCode.Residential
+    TRA = AlcsNfuTypeCode.Transportation_Utilities
+    UNU = AlcsNfuTypeCode.Unused
+
+
+class OatsToAlcsAgCapSource(Enum):
+    BCLI = AlcsAgCapSource.BCLI
+    CLI = AlcsAgCapSource.CLI
+    ONSI = AlcsAgCapSource.On_site
+
+
+class OatsToAlcsAgCap(Enum):
+    P = AlcsAgCap.Prime
+    PD = AlcsAgCap.Prime_Dominant
+    MIX = AlcsAgCap.Mixed_Prime_Secondary
+    S = AlcsAgCap.Secondary
+    U = AlcsAgCap.Unclassified
+
+
+@inject_conn_pool
+def process_alcs_application_prep_fields(conn=None, batch_size=BATCH_UPLOAD_SIZE):
```

Comment:
nitpick: function is pretty large, breakdown?

Reply:
Unfortunately, 90% of this function is python-prettier to break a one-line function call into 7 lines. Hopefully, this version will make it easier to read